authors/apps/articles/tests/test_rating_article.py

```diff
@@ -0,0 +1,96 @@
+"""
+test module for rating an article
+"""
+from django.urls import reverse
+from authors.apps.articles.tests.base_class import ArticlesBaseTest
+from ..models import Article
+from .test_data import valid_article, updated_article
```

Comment:
Hey @wasibani-roy, thanks for the good work, however on line 7 you import `valid_article` and `updated_article` but it seems you don't use them in this file.

Reply:
Worked on them